tests/config.rs

```diff
@@ -23,7 +23,6 @@ fn generate_config_with(eol: &str) {
 
     write!(stdin, "email@example.com{}", eol).unwrap();
     write!(stdin, "apikeythisissecretandlong{}", eol).unwrap();
-    write!(stdin, "{}", eol).unwrap();
```

Comment:
just interested in why this was removed

Reply:
this was a newline added when the old UX had folks fill out three fields when running `wrangler config` (the last field was API token, which was just an empty string when email + global API key combo was used)